.changeset/good-mayflies-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/search-text-input': minor
+---
+
+"feat(searchtext-input): add new isCondensed prop"
```

Comment:
Add `isCondensed` prop that when set to true, condenses the input height and font size.

Reply:
Added!